src/client/main.tsx

```diff
@@ -63,7 +63,7 @@ require.ensure([], require => {
   const appSettings = AppSettings.fromJS(config.appSettings, {
     visualizations: MANIFESTS,
     executorFactory: ({ name, cluster }: DataCube) => {
-      return Ajax.queryUrlExecutorFactory(name, "plywood", cluster.timeout);
+      return Ajax.queryUrlExecutorFactory(name, "plywood", cluster ? cluster.getTimeout() : 6000);
```

Comment:
This "6000" value is needed by "native" datasources which doesn't have cluster attached to them. Any ideas where this value should be defined? Should it be taken from Cluster.DEFAULT_TIMEOUT explicitly?

Reply:
1. For cluster mode the client timeout should be set to cluster.timeout + delta (e.g 5 seconds)
2. How to handle multiple clusters in the configuration file, max(DEFAULT, timeouts)?